### DISCUSSION TITLE
llhls "Could not find Origin for the stream"

### DISCUSSION THREAD

**Discussion by elliotd123** [OvenMediaLabs/OvenMediaEngine#1212] [Help] [answered]
I'm doing some testing to get OvenMediaEngine to work:

I'm trying to get LLHLS to work with an MPEG-TS Stream, but I keep getting "Could not find Origin for the stream" on the server logs.

The URL I am using is http://localhost:3333/app/stream/llhls.m3u8 (This is over SSH port forwarding, but I get the same result if I use the IP address and open the firewall).

Attached are the Server config and Log files. Renamed Server.xml because Github won't let me upload the xml directly.

[ovenmediaengine.log](https://github.com/AirenSoft/OvenMediaEngine/files/11384564/ovenmediaengine.log)
[Server.xml.txt](https://github.com/AirenSoft/OvenMediaEngine/files/11384566/Server.xml.txt)

**Comment by getroot** [comment, marked as answer]
In your logs I can't see the mpeg2-ts client's stream input. So there is no LLHLS playback as there are no streams in OME.

What client are you using to send the stream to OME?
If you are using ffmpeg see below.
https://airensoft.gitbook.io/ovenmediaengine/live-source/mpeg-2-ts-beta#publish

**Comment by elliotd123** [comment]
Thanks for a push in the right direction. I was able to resolve this particular issue by outputting the stream on a unicast address instead of a multicast address. Once I do that, the feed works.

I have another discussion about multicast mpeg-ts [here](https://github.com/AirenSoft/OvenMediaEngine/discussions/1211), so I will close out this discussion.
